db/migrate/20171109124132_add_enable_fields_for_optional_steps.rb

```diff
@@ -0,0 +1,7 @@
+class AddEnableFieldsForOptionalSteps < ActiveRecord::Migration[5.1]
+  def change
+    add_column :research_sessions, :where_when_enabled, :boolean, default: false
+    add_column :research_sessions, :expenses_enabled, :boolean, default: false
```

Comment:
We should check to see if any of the expenses fields have been filled in to set the field value

Reply:
@joelcarr You mean for each of the steps, for this migration alone? I'll take a look.

Reply:
Ok, this is rebased in now.